### ISSUE TITLE
Type 'string' is not assignable to type 'number'

### ISSUE THREAD

**Issue by leandomandtom** [FriendsOfReact/react-native-gradients#21]
When you are trying to set percentages as values there is a Typescript error:
`Type 'string' is not assignable to type 'number'.ts(2322)
RadialGradient.d.ts(7, 5): The expected type comes from property 'rx' which is declared here on type 'IntrinsicAttributes & { colorList: Color[]; x: number; y: number; rx: number; ry: number; }'`

```
<RadialGradient colorList={colorList} rx="50%" ry="50%" x="50%" y="50%" />
```

the types of the properties are just number:
`export const RadialGradient = ({ colorList, x, y, rx, ry }: {  colorList: Color[]; x: number; y: number; rx: number; ry: number}) => {`

**Comment by ZhenyaKu** [comment]
have the same issue

**Comment by yildizberkay** [comment]
Fixed #19
